app/boards/shields/waterfowl/waterfowl.keymap

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+/ {
+	keymap {
+		compatible = "zmk,keymap";
+
+		default_layer {
+/* QWERTY
+ *
+ * ,----------------------------------.                      ,----------------------------------.
+ * |   Q  |   W  |   E  |   R  |   T  |                      |   Y  |   U  |   I  |   O  |   P  |
+ * |------+------+------+------+------|                      |------+------+------+------+------|
+ * |   A  |   S  |   D  |   F  |   G  |                      |   H  |   J  |   K  |   L  |   ;  |
+ * |------+------+------+------+------|  ,-----.    ,-----.  |------+------+------+------+------|
+ * |   Z  |   X  |   C  |   V  |   B  |  |  2  |    |  3  |  |   N  |   M  |   ,  |   .  |   /  |
+ * `----------------------------------'  `-----'    `-----'  `----------------------------------'
+ *          ,-----.   ,--------------------.            ,--------------------.   ,-----. 
+ *          |  1  |   | DEL | SPACE | TAB  |            |  ESC  | BS | ENTER |   |  4  |	
+ *          `-----'   `--------------------'            `--------------------'   `-----'
+ */
+			bindings = <
+	&kp Q       &kp W 	   &kp E       &kp R       &kp T            &kp Y   &kp U        &kp I       &kp O      &kp P
+	&mt LGUI A  &mt LALT S &mt LCTRL D &mt LSHFT F &kp G            &kp H   &mt LSHFT J  &mt LCTRL K &mt LALT L &mt LGUI SEMI
+	&kp Z       &kp X      &kp C       &kp V       &kp B       		&kp N   &kp M        &kp COMMA   &kp DOT    &kp FSLH
+	&kp N1      &lt 3 DEL  &lt 1 SPACE &kp TAB     &kp N2      		&kp N3  &kp ESC      &kp BSPC    &lt 2 RET  &kp N4
```

Comment:
Some tab/space mixup between left/right blocks in the layer definitions as well, it throws off the alignment when tabstop is set to 8 (Github default)